frontend/app/views/spree/checkout/_summary.html.erb

```diff
@@ -12,7 +12,7 @@
         <% order.line_item_adjustments.nonzero.promotion.eligible.group_by(&:label).each do |label, adjustments| %>
           <tr class="total">
             <td><%= label %></td>
-            <td><%= Spree::Money.new(adjustments.sum(&:amount), currency: order.currency).to_html %></td>
+            <td><%= Spree::Money.new(adjustments.sum(&:amount), currency: order.currency).to_html %> <%= content_tag(:span, '', class: 'glyphicon glyphicon-minus-sign delete-coupon-button', id: 'delete-promotion-coupon', value: order.line_item_adjustments.first.source.promotion.code) %></td>
```

Comment:
@WitekL maybe let's split this into multi-line for greater readability

Reply:
`order.line_item_adjustments.first.source.promotion.code` - maybe it would be better to move this into some method?

Reply:
`order.line_item_adjustments.first` shouldn't it be `adjustments.first` ?

Reply:
I used `order.line_item_adjustments` because in this segment the line_item_adjustments are being displayed. Below, in the second span, I used `order.shipment_adjustments` to differentiate between both of them.